src/mitigation.c

```diff
@@ -54,18 +56,37 @@ void mitigation_add_resource(struct mitigation *m, const char *name, const char
 
 	r->resource = res;
 
-	list_append(&m->resources, &r->list_node);
+	// resources that have the attribute fallback will be added to a seperate list
+	if(fallback)
```

Comment:
The fallback attribute isn't presented as a boolean, but behaves as such.

fallback="true"/"false" and the relevant code is what I would expect here.